x-pack/plugins/ml/public/file_datavisualizer/components/summary/summary.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+
+import {
+  EuiDescriptionList,
+  EuiTitle,
+} from '@elastic/eui';
+
+export function Summary({ results }) {
+  const items = createDisplayItems(results);
+
+  return (
+    <React.Fragment>
+      <EuiTitle size="s">
+        <h3>Summary</h3>
+      </EuiTitle>
+      <EuiDescriptionList
+        type="column"
+        listItems={items}
+        style={{ maxWidth: '400px' }}
```

Comment:
Worth having this in a `scss` file?

Reply:
i believe specifying the `style` like this is the preferred way for eui components.
This is how it is done in the eui example